### ISSUE TITLE
Line wrapping wraps inside parentheses of no-arg method calls if line is very long

### ISSUE THREAD

**Issue by Hubry** [Vineflower/vineflower#67]
On field and variable declarations where the type combined with the initializer are very long (usually involving generics), the decompiler tends to wrap them like this:
```java
    private static final Function<Entry<Property<?>, Comparable<?>>, String> PROPERTY_ENTRY_TO_STRING_FUNCTION = new Function<Entry<Property<?>, Comparable<?>>, String>(
        
    ) {
```
This is not quite ideal and could be wrapped before the `=`, but wrapping inside `()` probably should never happen even if that's the only way to shorten the line.
Using Mojmap there are 25 instances of this, easily found with the regex `\(\n +\n +\)`.

**Comment by jpenilla** [comment]
Fixed by #362, wrapping after `=` is also an interesting idea to investigate

**Comment by jaskarth** [comment]
Thanks for the shout, and for the report!
